.ci/tweet-releasenotes.sh

```diff
@@ -32,17 +32,32 @@ CS_RELEASE_VERSION="$(getCheckstylePomVersion)"
 echo CS_RELEASE_VERSION="$CS_RELEASE_VERSION"
 
 cd .ci-temp/checkstyle
-LATEST_RELEASE_TAG=$(curl -s https://api.github.com/repos/checkstyle/checkstyle/releases/latest \
-                       | jq ".tag_name")
-echo LATEST_RELEASE_TAG="$LATEST_RELEASE_TAG"
+
+curl https://api.github.com/repos/checkstyle/checkstyle/releases \
+    -H "Authorization: token $GITHUB_TOKEN" \
+    -o /var/tmp/cs-releases.json
+
+LATEST_RELEASE=$1
+TARGET_RELEASE_INDEX=$(cat /var/tmp/cs-releases.json | \
+    jq '[.[].tag_name] | to_entries | .[] | select(.value=="checkstyle-$LATEST_RELEASE") | .key')
+echo "$TARGET_RELEASE_INDEX"
+PREVIOUS_RELEASE_INDEX=$(($TARGET_RELEASE_INDEX+1))
+echo "$PREVIOUS_RELEASE_INDEX"
+
+START_REF=$(cat /var/tmp/cs-releases.json | jq -r ".[$TARGET_RELEASE_INDEX].tag_name")
+END_REF=$(cat /var/tmp/cs-releases.json | jq -r ".[$PREVIOUS_RELEASE_INDEX].tag_name")
```

Comment:
END_REF should be taken by TARGET_RELEASE_INDEX.

builder goes from start(old release) to end (target release).

Reply:
Changes Done